password_security/models/res_users.py

```diff
@@ -205,3 +205,13 @@ def _set_encrypted_password(self, uid, pw):
 
         self.write({"password_history_ids": [(0, 0, {"password_crypt": pw})]})
         return res
+
+    def action_reset_password(self):
+        """Disallow password resets inside of Minimum Hours"""
+        if self.env.context.get("install_mode"):
+            return
```

Comment:
Maybe negate this and indent the following lines to remove the uncovered return? Falling through to the parent is fine, because it has its own install_mode checks.